sdk/identity/Azure.Identity.BrokeredAuthentication/src/SharedTokenCacheCredentialBrokerOptions.cs

```diff
@@ -32,7 +32,7 @@ public SharedTokenCacheCredentialBrokerOptions(TokenCachePersistenceOptions toke
 
         private void AddBroker(PublicClientApplicationBuilder builder)
         {
-            builder.WithBrokerPreview();
+            builder.WithBroker();
```

Comment:
There should be an overload to this method in `Microsoft.identity.Client.Broker` package, that takes in BrokerOptions. 

`.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows))`

Reply:
We went for an API that enables the broker on each platform individually, as each platform has a different redirect uri. Only Win is supported now.

Reply:
Change done